.themes/default/source/arcs.html

```diff
@@ -15,8 +15,6 @@ <h3 class="arc-title"><a name="{{ rp.name | slugify }}" href="#{{ rp.name | slug
       {% include rp.html rp=arc_rp %}
     {% endfor %}
     </ul>
-  {% else %}
-    {% include rp.html rp=rp %}
```

Comment:
It is an index of all arcs, why would you put non arc'ed rps on the page? Leads to a lot of clutter.